demo-app/src/app/app.component.html

```diff
@@ -1,4 +1,27 @@
-<div class="container" fxLayout="row" fxLayoutAlign="space-between center">
+<app-navigation></app-navigation>
+
+<div class="container" fxLayout="column" fxLayoutAlign="space-between center">
```

Comment:
Do you need column here. Unless it is necessary Please avoid using column since div itself is block element it will be displayed in column only

Reply:
Use proper formate for writing HTML code, class should be coming at last. refer wiki under FORMATTING.